.github/actions/environment/action.yml

```
@@ -42,7 +42,8 @@ runs:
           maui-android \
           ${{ runner.os == 'macOS' && 'maui-ios maui-maccatalyst maui-windows maui-tizen' || '' }} \
           ${{ runner.os == 'Windows' && 'maui-windows' || '' }} \
-          --temp-dir "${{ runner.temp }}" --from-rollback-file rollback.json
+          --temp-dir "${{ runner.temp }}" --from-rollback-file rollback.json \
+          --skip-sign-check
```

```suggestion
          --skip-sign-check
          # Skipping signing check above because: https://github.com/dotnet/maui/issues/9573
```

Unrelated but didnt' we remove rollback.json ?

---

We did on the `feat/4.0.0` branch

---

I wonder if this has something to do with the `--from-rollback-file` parameter. We weren't experiencing these problems on the `feat/4.0.0` branch where that parameter had been removed.